extensions/oidc/runtime/src/main/java/io/quarkus/oidc/AuthenticationContext.java

```diff
@@ -0,0 +1,34 @@
+package io.quarkus.oidc;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.TYPE;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation which can be used to enforce required authentication level for the endpoint classes and methods.
+ * <a href="https://datatracker.ietf.org/doc/rfc9470/">OAuth 2.0 Step Up Authentication Challenge Protocol</a>
+ * is initiated when the access token does not have sufficient authentication strength.
+ */
+@Target({ TYPE, METHOD })
+@Retention(RetentionPolicy.RUNTIME)
+public @interface AuthenticationContext {
+
+    /**
+     * By default, max age is not checked.
+     */
+    long NO_MAX_AGE = -1;
+
+    /**
+     * Required `acr` (Authentication Context Class Reference) claim values.
```

Comment:
```suggestion
     * Required ACR ('acr') claim values.
```

Proposing to use an `ACR` shortcut which is proposed above